backend/enhanced_guide_generator.py

```
@@ -0,0 +1,796 @@
+        
+        enhanced_guide = await self._enhance_for_luxury_standards(
+            base_guide, destination, start_date, end_date, hotel_info, preferences, extracted_data
+        )
+        
+        return enhanced_guide
+    
+    def _create_fallback_guide(self, destination: str, start_date: str, end_date: str, hotel_info: Dict, preferences: Dict) -> Dict:
+        """Create comprehensive fallback guide when API fails"""
+        
+        # Calculate trip duration
+        from datetime import datetime
```

Ruff (0.13.1)

Redefinition of unused `datetime` from line 8

Remove definition: `datetime`

(F811)